File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/python/java/p02880.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        for (int i = 1; i < 10; i++) {
            for (int j = 1; j < 10; j++) {
                if (i * j == N) {
                    System.out.println("Yes");
                    return;
                }
            }
        }
        System.out.println("No");
    }
}
```

This Java code does the same thing as your Python code. It reads an integer from the user, then it checks if there is a pair of integers (i, j) such that i * j equals the input number. If such a pair exists, it prints "Yes" and exits the program. If no such pair exists, it prints "No".

File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/java/python/p02880.py
Here is the Python equivalent of the Java code:

```python
N = int(input())
f = False
for A in range(1, 10):
    if N % A == 0 and N / A < 10:
        f = True
ret = "Yes" if f else "No"
print(ret)
```

This Python code does the same thing as the Java code. It reads an integer from the user, checks if it is divisible by any number from 1 to 9 and if the quotient is less than 10, and then prints "Yes" if the condition is met, otherwise it prints "No".
